apps/api/R/models.R

```diff
@@ -1,31 +1,52 @@
 library(dplyr)
 
-#' Retrieve the details of a particular version of a model
-#' @param name Model name (character)
-#' @param revision Model version/revision (character)
+#' Retrieve the details of a PEcAn model, based on model_id
+#' @param model_id Model ID (character)
 #' @return Model details
 #' @author Tezan Sahu
-#* @get /
-getModels <- function(model_name="all", revision="all", res){
+#* @get /<model_id>
+getModel <- function(model_id, res){
```

Comment:
Yeah, so this portion is related to the "Visibility Details" for which I had created an issue earlier: #2639 (here, it is mentioned for workflows & runs, but I think this can be extended to models & sites as well)
At a given point in time, the user will be interacting with the API server on a particular machine, so in that case, if the `HOST_ONLY` flag is `TRUE`, the results for the search will contain only the models/sites available o the host machine that the user is interacting with & if `FALSE` (ie, the default value of `HOST_ONLY`), all available versions & models based on the search will be returned.

This portion is left for implementation towards some later part in this project. Hope it sounds fine.